src/pages/profile/components/Review/index.tsx

```diff
@@ -0,0 +1,25 @@
+import ReviewContent from './ReviewContent';
+
+interface Review {
+  score: number;
+  content: string;
+}
+interface ReviewProps {
+  reviews: Review[];
+}
+const Review = ({ reviews }: ReviewProps) => {
+  return (
+    <div>
+      <span className='font-bold'>매너 리뷰</span>
+      <div className='py-1'>
```

Comment:
`gap` 속성은 `display: flex`, `display: grid`인 경우 동작하기 때문에 적용되지 않았을 거예요.
아래처럼 작성하시면 `mb-2`는 지우셔도 될 것 같아요!

```suggestion
      <div className='flex flex-col gap-2 py-1'>
```